examples/cra-kitchen-sink/package.json

```diff
@@ -39,6 +39,9 @@
     "@storybook/components": "^3.3.0-alpha.4",
     "@storybook/react": "^3.3.0-alpha.4",
     "babel-jest": "^21.2.0",
+    "enzyme": "^3.2.0",
```

Comment:
We need to have a single enzyme instance across the codebase, so it shouldn’t be added to package dependencies

Reply:
i know. but i figured an "example project" should include the necessary dependencies? its a lot more difficult to understand if i need to look outside of the project for its code to work. thats why i included enzyme setup there

Reply:
OK, makes sense